src/keas/kmi/testclient.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
This should go after the block comment, right above the `__docformat__` assignment (which can probably be thrown away).